api-reference/10 UI Components/dxFileManager/1 Configuration/onItemMoving.md

```diff
@@ -5,34 +5,170 @@ default: null
 ---
 ---
 ##### shortDescription
-<!-- Description goes here -->
+A function that is executed before a file or folder is moved.
 
 ##### param(e): Object
-<!-- Description goes here -->
+Information about the event.
 
 ##### field(e.cancel): Boolean | Promise<void>
-<!-- Description goes here -->
+Allows you to cancel the file or folder move process.
 
 ##### field(e.component): dxFileManager
-<!-- Description goes here -->
+The UI component's instance.
 
 ##### field(e.destinationDirectory): FileSystemItem
-<!-- Description goes here -->
+The directory where a file is moved to.
```

Comment:
Maybe better "The directory where a file is being moved to" since event occurrs before move opearion.